.github/ISSUE_TEMPLATE/maintainers-only.yml

```diff
@@ -0,0 +1,38 @@
+name: Maintainers only
+description: For maintainers only
+labels: ['status:requirements', 'priority-5-triage', 'needs-discussion']
+body:
+  - type: markdown
+    attributes:
+      value: '# This form is for Renovate maintainers only.'
+
+  - type: markdown
+    attributes:
+      value: |
+        **Stop!** This form is for the Renovate maintainers only.
+        If you're a user of Renovate, please create a GitHub Discussion.
+        Please go back a step and select _Start a discussion_ instead of Issue.
+
+  - type: textarea
+    id: describe-proposed-changes
+    attributes:
+      label: Describe the proposed change(s).
+      description: 'Do not report any security concerns here. Email renovate-disclosure@mend.io instead.'
+    validations:
+      required: true
+
+  - type: textarea
+    id: descibe-why-we-need-want-these-changes
+    attributes:
+      label: Describe why we need/want these change(s).
+      description: 'Do not report any security concerns here. Email renovate-disclosure@mend.io instead.'
```

Comment:
Do we need this twice?

Reply:
I want to have two fields in the form:

1. The _what_ of the change.
2. The _why_ of the change (background info).

Do you want me to drop the second _why_ field, or do you want me to drop the second security concerns warning?